operatorcourier/flatten.py

```diff
@@ -4,8 +4,10 @@
 from typing import Dict, Tuple
 from shutil import copyfile
 import semver
-from operatorcourier import identify
-from operatorcourier import errors
+from operatorcourier.identify import get_operator_artifact_type
+from operatorcourier.errors import OpCourierBadBundle, OpCourierBadYaml
+from operatorcourier.verified_manifest \
+    import is_manifest_folder, get_package_csv_info_from_root
```

Comment:
This seems like a little bit of an anti pattern. If `Verified_Manifest` is a class, shouldn't only class functions come from this package? If they aren't defined as part of this class (which would imply they were called by an instance of that type) then they probably shouldn't be in that module